### ISSUE TITLE
Where can I download a build of armory for Windows?

### ISSUE THREAD

**Issue by Mitra-M** [armory3d/armory#485]
I saw [this page](https://github.com/armory3d/armory/releases) ,but only the source code is there.

Where can I download a build of armory (exe file) for Windows?

**Comment by Simonrazer** [comment]
you need to be a patreon, and you can become one here: https://www.patreon.com/armory (min 10$/month)
or one-time 50$ here http://armory3d.org/download.html

**Comment by Mitra-M** [comment]
aha, thanks for the reply.

I'm not too familiar with Blender, but I know Blender also has a built-in game engine. Please explain **briefly** what is the major difference between Armory and BGE?

**Comment by Simonrazer** [comment]
BGE is ages old = relatively bad graphics. Armory is modern, so I looks very nice and has builtin Global illumination, can export to everything (or will eventually), uses Cycles material nodes and logic nodes. BGE uses Blender Internal materials (kind of), I have only seen windows + mac exports from it, but it uses very easy to understand logic bricks. 
Also Armory has supports better Cloth and softbody sim.

**Comment by AFWSI** [comment]
Yeah, I agree with Simonrazer. A lot of people are trying to save the BGE ,but it would be nice if Armory would become the official game engine for blender. Mitra-M are you on Blenderartist.org?

**Comment by Mitra-M** [comment]
@AFWSI , No I'm not .
I just check out free game engines to choose the best. (the best  for me)

**Comment by AFWSI** [comment]
@Mitra-M OK, if you decide to join send me a message, I’m AFWS on there.

I’ve also been checking every game engine known to man. I’ve got a project I’m really fixin to put a lot of time into ,but I haven’t decided on which engine I want to use for it yet. I played around with Unreal4, Godot, BGE, UPBGE, and now Armory. Prolly gonna use the one easiest for logic/coding, cause my coding skills aren’t that great.

**Comment by Mitra-M** [comment]
@AFWSI   OK, Thanks.

**Comment by Mitra-M** [comment]
@AFWSI , can you please tell me (by email please ) what is the result of your reviews about game engines? Did you finally choose Armory? 
Please reply me with this email :  media4dco@gmail.com

Thanks.

**Comment by AFWSI** [comment]
I haven’t decide which I’m gonna use yet. Prolly will end up being Armory or Godot. The learning curve for Unreal4 is a little much for me and BGE is to outdated.

**Comment by Mitra-M** [comment]
Thank you for your reply.

Honestly, I'm going to make a 2D/3D application for live TV broadcasting.
(crawl/roll text, logo , animation ... on live show like news channels)

that need to a Real-time rendering core+ interactive UI at runtime ,
 but since the 3D rendering core is out of my knowledge, I'm looking for an
easy open source game engine that I can modify it for my purpose.

I'm also tried : Cryengine , Xenko, Unreal4  Godot, BGE

  I was able to compile and test all of the above engines. (BGE was the
easiest for me, but unfortunately interaction with UI is not possible in
it.)

now , I want to try Armory , but Its autors have not guided and I can not
compile that for Win10 .

 I realized (from your posts on Armory forums) that you have build Armory ,
Can you please guide me to compile it?

(sorry for my bad English)
